Cargo.toml

```
@@ -20,7 +20,7 @@ serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
 node-bindgen = "5.1.0"
 flate2 = "1.0.24"
-fluvio = { version = "0.18.1", features = ["admin"] }
+fluvio = { features = ["admin"], git = "https://github.com/nacardin/fluvio.git", rev = "434a089e55826fe1f034eb5a4e061630afe8733d" }
```

maybe the tag instead?

```suggestion
fluvio = { features = ["admin"], git = "https://github.com/nacardin/fluvio.git", tag = "v0.10.8-dev-1" }
```

---

there was a bug in `v0.10.8-dev-1`, will update to `v0.10.8-dev-2` when ready